lib/trento/operations/host_policy.ex

```diff
@@ -23,7 +25,7 @@ defmodule Trento.Operations.HostPolicy do
         } = host,
         _
       )
-      when operation in [:saptune_solution_apply, :saptune_solution_change] do
+      when operation in HostOperations.values() do
```

Comment:
the same here, this is just for saptune operations, not most probably all host operations

Reply:
0b6e4bd35f1ff67eb3a01e0f436f87e89dbf4bd8